recipe/meta.yaml

```diff
@@ -7,10 +7,10 @@ package:
 source:
     fn: cell_tree2d-{{ version }}-.tar.gz
     url: https://github.com/NOAA-ORR-ERD/cell_tree2d/archive/v{{ version }}.tar.gz
-    sha256: dbdb0f6f3b68536f73b9db79c1199753504fbf58397551b4c9bf210aa364cbce
+    sha256: 4a5759951f67a3ff199137c87bb565b1897541130de5a4a2067acb22b0f39bae
 
 build:
-    number: 0
+    number: 1
```

Comment:
No need to bump the build number when building a new version. Although I am getting confused with all those releases of `cell_tree2d` with a few minutes... Maybe we should improve the CI testing there to catch things early.